### PR TITLE
Do registration for media upgrade cases at system_prepare module

### PR DESCRIPTION
**Pull request by tinawang123**
Do registration for media upgrade cases at system_prepare module.
As we need do regression test after media upgrade, we should register
the modules before regression

Changed the check conditions, as we only can do register at regression job group. The rule is very strictly. I changed it to use setting. It can do register wherever users want. It should not check scc_addons, as some module is registered by default. So I remove this condition. And the upgrade path from sles12+ to sles15+. It will add some modules by default. 

- Related ticket:https://progress.opensuse.org/issues/98195
- Verification run: https://openqa.suse.de/tests/7068280
- https://openqa.suse.de/tests/7097714#

### CODE REVIEW

**Comment by hjluo** [comment]
LGTM

**Comment by hjluo** [comment]
LGTM

**Review by chcao — APPROVED** [review]
LGTM

File: tests/console/system_prepare.pm [hunk 1]
```diff
@@ -55,7 +55,11 @@ sub run {
             add_suseconnect_product(get_addon_fullname('desktop'), undef, undef, undef,                             300, 1);
             add_suseconnect_product(get_addon_fullname('we'),      undef, undef, "-r " . get_var('SCC_REGCODE_WE'), 300, 1);
         }
-        my $myaddons = get_var('SCC_ADDONS');
+        my $myaddons = get_var('SCC_ADDONS', '');
+        $myaddons .= "dev,lgm,wsm" if (is_sle('<15', get_var('ORIGIN_SYSTEM_VERSION')) && is_sle('15+'));
```
Comment (lemon-suse) [inline tests/console/system_prepare.pm:59]: if there is a SDK in SCC_ADDONS, you haven't deal with such situation.

Comment (lemon-suse) [inline tests/console/system_prepare.pm:59]: if there is a SDK in SCC_ADDONS, you haven't deal with such situation.

Comment (tinawang123) [reply tests/console/system_prepare.pm:59]: sdk       => is_sle('15+') ? 'sle-module-development-tools' : 'sle-sdk',
It did register at lib/registration.pm file

**Review by lemon-suse — APPROVED** [review]

**Pull request merged**